pegasus/sites.v3/code.org/public/learn/index.haml

```diff
@@ -120,8 +120,8 @@ style_min: false
   - tutorial[:string_detail_platforms]              = hoc_s(prefix + "string_detail_platforms")
   - tutorial[:string_detail_programming_languages]  = hoc_s(prefix + "string_detail_programming_languages")
 
-- sort_by_popularity = request.site == 'code.org'
-- sort_by_popularity &&= [false, "pre-hoc"].include?(DCDO.get("hoc_mode", CDO.default_hoc_mode))
+- hoc_mode = DCDO.get("hoc_mode", CDO.default_hoc_mode)
+- sort_by_popularity = [false, "post-hoc"].include?(hoc_mode) || (request.site == 'code.org' && hoc_mode == 'pre-hoc')
```

Comment:
This changes the behavior when `request.site` is `csedweek.org` and `hoc_mode` is `false`. Before, `sort_by_popularity` would be `false` in that scenario; now it's `true`.  I'm pretty sure the fix is:

```suggestion
- sort_by_popularity = ("post-hoc" == hoc_mode) || (request.site == 'code.org' && [false, 'pre-hoc'].include? hoc_mode)
```

Reply:
makes sense, incorporated that change into new method